ostorlab_insecure_flutter_app/lib/bugs/command_exec.dart

```diff
@@ -13,16 +13,21 @@ class CommandExec extends BugRule {
 
   /// Trigger the [BugRule]
   @override
-  Future<void> run() async {
-    TextEditingController fileController = TextEditingController(text: "/sdcard/document.pdf");
-    TextEditingController domainController = TextEditingController(text: "www.ostorlab.co");
+  Future<void> run(String user_input) async {
+    TextEditingController fileController =
+        TextEditingController(text: "/sdcard/document.pdf");
+    TextEditingController domainController =
+        TextEditingController(text: "www.ostorlab.co");
```

Comment:
Replace the text controller with the input. You can check the iOS code.

Reply:
[c8a9079](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/22/commits/c8a90794062425fbef2446864eb4c9584a01e667)